contentcuration/contentcuration/frontend/channelEdit/vuex/contentNode/actions.js

```diff
@@ -35,6 +36,161 @@ export function loadChildren(context, { parent, channel_id }) {
   });
 }
 
+/**
+ * Load a whole chain of parent nodes of a node.
+ */
+export async function loadParents(context, { id, channel_id }) {
```

Comment:
I also added a loadAncestors method if you'd like to adapt it https://github.com/learningequality/studio/blob/vue-refactor/contentcuration/contentcuration/frontend/channelEdit/vuex/contentNode/actions.js#L38

Reply:
Oh good, this looks much more elegant for a tree structure. It would be great to use this instead of recursion.

@jayoshih 
I've checked our IndexedDB tree table data in more detail and can't find `lft` field so I am not sure how `max_lft: node.sort_order` part of where clause is supposed to work. Do I need to migrate IndexedDB somehow or do we rather use `sort_order` as left value in a tree or have I missed something else?

And another question - is there a reference for this where clause format? So far I've found only https://dexie.org/docs/WhereClause/WhereClause.

Reply:
In the viewset, `lft` has been mapped to `sort_order` [here](https://github.com/learningequality/studio/blob/vue-refactor/contentcuration/contentcuration/viewsets/tree.py#L71), and the min/max values are part of the [filter](https://github.com/learningequality/studio/blob/vue-refactor/contentcuration/contentcuration/viewsets/tree.py#L24). Eventually, we'll be using a more decoupled way to query for ancestors, but changes related to that are in the master branch right now

Reply:
Oh good, I haven't found those. Thanks a lot.